API/API.csproj

```
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="12.0.0" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
   </ItemGroup>
```

Referencia ao AutoMapper deve estar na AppServices, e o package de DependencyInjection do AutoMapper, depende do AutoMapper original, portanto não há a necessidade de usar ambos.
```suggestion
  <ItemGroup>
    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
  </ItemGroup>
```